CLASSES/class_person.py

```diff
@@ -0,0 +1,92 @@
+class Person:
+    max_health = 100
+    base_stats = ("attack", "defense", "intelligence", "health")
+
+    def __init__(self, features: dict):
+
+        self.features = features
+        self.first_name = self.features["first_name"]
+        self.last_name = self.features["last_name"]
```

Comment:
Podoba mi się, że słownik ma klucze z danymi atrybutami instancji. To jest czytelne